src/corona_hakab_model/consts.py

```diff
@@ -29,7 +29,7 @@ class Consts(NamedTuple):
 
     # attributes and default values:
 
-    total_steps: int = 350
+    total_steps: int = 40
```

Comment:
switch this back to a higher number before merging

Reply:
oops :laughing: